### ISSUE TITLE
Search Stream Logging: Request Id Empty?

### ISSUE THREAD

**Issue by aukevin** [googleads/google-ads-dotnet#283]
I recently noticed that with my `TraceUtilities` settings, `Search` requests include Request Id information while `SearchStream` requests do not. This behavior feels inconsistent and can pose some friction when providing information to Support.

For example, when I try to get the `Information` level summary logs:

```
TraceUtilities.Configure(TraceUtilities.SUMMARY_REQUEST_LOGS_SOURCE, listener, System.Diagnostics.SourceLevels.Information);
```

The Search logs look something like:

[2021-05-04 12:00:40Z] - Request made: Host: https://googleads.googleapis.com, Method: /google.ads.googleads.v6.services.GoogleAdsService/Search, ClientCustomerID: [Redacted],  RequestID: [Redacted, but present], IsFault: False, FaultMessage: 

While the SearchStream logs from the same execution look like:

[2021-05-04 12:00:40Z] - Request made: Host: https://googleads.googleapis.com, Method: /google.ads.googleads.v6.services.GoogleAdsService/SearchStream, ClientCustomerID: [Redacted],  RequestID: , IsFault: False, FaultMessage: 

Does anyone have a work-around for this issue?

-- Kevin

**Comment by aukevin** [comment]
Hi, I'm still having trouble with this. Is there another way to get tracking information on `SearchStream` requests?

**Comment by aukevin** [comment]
I'll amend this to say that _sometimes_ I get the RequestId. I can't seem to figure out what triggers it to appear or not.

**Comment by aukevin** [comment]
@AnashOommen This has come up again when talking with support on the Google Ads Forum. There is no request id to provide support with in many of my `SearchStream` calls, especially in failure cases. Help on this would be much appreciated.

**Comment by aukevin** [comment]
I enabled some more detailed logging and the logs without Request Id look something like this when more verbose logging is enabled:

[2021-06-08 21:28:38Z] - 
---------------BEGIN API CALL---------------

Request
-------

Method Name: /google.ads.googleads.v6.services.GoogleAdsService/SearchStream
Host: https://googleads.googleapis.com
Headers: {
  "x-goog-api-client": "gl-dotnet/3.1.0 gapic/6.1.0 gax/3.2.0+002f6637d17e4170f5db7ec7ca6f0163e6df1481 grpc/2.34.1",
  "developer-token": "REDACTED",
  "login-customer-id": "//** OMITTED **//"
}

{ "customerId": "//** OMITTED **//", "query": "SELECT ad_group.cpc_bid_micros, ad_group.cpm_bid_micros, ad_group.effective_target_cpa_micros, ad_group.final_url_suffix, ad_group.id, ad_group.name, ad_group.status, ad_group.targeting_setting.target_restrictions, ad_group.tracking_url_template, ad_group.type, ad_group.url_custom_parameters, campaign.id, campaign.name, metrics.average_cpc, metrics.clicks\nFROM ad_group\nWHERE campaign.id IN (//** OMITTED **//)\nORDER BY ad_group.id ASC", "summaryRowSetting": "NO_SUMMARY_ROW" }

Response
--------
Headers: {
  "content-disposition": "attachment",
  "date": "Tue, 08 Jun 2021 21:28:38 GMT",
  "alt-svc": "h3-29=\":443\"; ma=2592000,h3-T051=\":443\"; ma=2592000,h3-Q050=\":443\"; ma=2592000,h3-Q046=\":443\"; ma=2592000,h3-Q043=\":443\"; ma=2592000,quic=\":443\"; ma=2592000; v=\"46,43\""
}

{ "results": [ //** RESULTS OMITTED **// ], "fieldMask": "adGroup.cpcBidMicros,adGroup.cpmBidMicros,adGroup.effectiveTargetCpaMicros,adGroup.finalUrlSuffix,adGroup.id,adGroup.name,adGroup.status,adGroup.targetingSetting.targetRestrictions,adGroup.trackingUrlTemplate,adGroup.type,adGroup.urlCustomParameters,campaign.id,campaign.name,metrics.averageCpc,metrics.clicks" }
----------------END API CALL----------------

**Comment by AnashOommen** [comment]
This issue is fixed in v11.0.0.
